test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/SortTests.cs

```diff
@@ -90,13 +91,82 @@ public async Task GivenMoreThanTenPatients_WhenSearchedWithSortParamWithHyphen_T
             await ExecuteAndValidateBundle($"Patient?_tag={tag}&_sort=-_lastUpdated", false, patients.Reverse().Cast<Resource>().ToArray());
         }
 
+        // uncomment only when db cleanup happens on each run, otherwise the paging might cause expected resources to not arrive
+        /*[Fact]
+        [HttpIntegrationFixtureArgumentSets(dataStores: DataStore.SqlServer)]
+        public async Task GivenQueryWithNoFilter_WhenSearchedWithSortParam_ThenResourcesAreReturnedInAscendingOrder()
```

Comment:
Should we add tests to verify descending order too?

Reply:
@eladiw Do you have time to add the DESC tests as well? it looks pretty quick

Reply:
@eladiw I added these tests